lms/tasks/email_digests.py

```diff
@@ -119,9 +138,13 @@ def send_instructor_email_digests(
                         **self.request.kwargs,
                         "h_userids": list(err.errors.keys()),
                     },
-                    countdown=retry_countdown,
+                    countdown=_retry_countdown(self.request),
                 )
             except Exception as exc:  # pylint:disable=broad-exception-caught
                 LOG.exception(exc)
                 report_exception(exc)
-                self.retry(countdown=retry_countdown)
+                self.retry(countdown=_retry_countdown(self.request))
+
+
+def _retry_countdown(celery_request):
```

Comment:
If we move this over to celery.py later we probably want to add a couple kwargs with these defaults.